Helper/Data.php

```diff
@@ -72,19 +68,18 @@ public function log($value)
 
     public function createActivity($storeId, $type)
```

Comment:
Please move this to a separate helper, maybe `ActivityHelper`?

Reply:
Moved.